cpu/k20/include/mk20dx128.c

```diff
@@ -0,0 +1,1000 @@
+/* Teensyduino Core Library
```

Comment:
a .c file in the include path? Seems like this file is redundant with the `startup.c` and also with a lot of things defined in the cortexm_common part.